src/saleor_app/endpoints.py

```diff
@@ -35,7 +35,13 @@ async def install(
     if hasattr(request.app, "webhook_router"):
         for event_type, _ in request.app.webhook_router.http_routes.items():
             events[request.url_for("handle-webhook")].append(
-                (event_type, request.app.webhook_router.http_routes_subscriptions.get(event_type)))
+                (
```

Comment:
Shouldn't we iterate on `http_routes` instead of `http_routes.items()` and remove `, _` from forloop if we are not using dict item's value here?